cursorless-talon-dev/src/cursorless_dev_vscode.py

```diff
@@ -0,0 +1,28 @@
+from talon import Context, Module
+
+mod = Module()
+ctx = Context()
+
+ctx.matches = r"""
+app: vscode
+win.title: /cursorless/
+"""
+
+
+mod.list(
+    "cursorless_launch_configuration", "A launch configuration for Cursorless vscode"
+)
+
+ctx.lists["user.cursorless_launch_configuration"] = {
```

Comment:
use a `.talon-list`? i find i end up with one eventually, so may as will start there.

Reply:
Ah interesting idea. Haven't actually played with those yet. I'll try that

Reply:
I agree but that would make Cursorless 0.4 only

Reply:
This is just for Cursorless contributors. I'm comfortable requiring them to be on 0.4

Reply:
Same here

Reply:
For future travellers, we already made Cursorless require Talon 0.4, see #1880.